geomesa-accumulo/geomesa-accumulo-tools/pom.xml

```diff
@@ -159,7 +159,7 @@
                         <argument>org.locationtech.geomesa.accumulo.tools.AccumuloRunner</argument>
                         <argument>help</argument>
                         <argument>--autocomplete-function</argument>
-                        <argument>${project.build.directory}/autocomplete.sh,geomesa</argument>
+                        <argument>${project.build.directory}/autocomplete.sh,geomesa-accumulo</argument>
```

Comment:
Where did you land on the autocomplete.sh?

Reply:
still there. this fixes it for accumulo